### ISSUE TITLE
Make navigation accessible

### ISSUE THREAD

**Issue by Kriesse** [coworkingfestival/coworkingfestival.github.io#15]
The main navigation is currently not accessible for keyboard users or vision-impaired.

- [ ] use semantic elements (e.g. button instead of a for the toggle button)
- [ ] add WAI ARIA attributes to menu toggles and navigation items
